client-admin/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Gulp v3 stops us from upgrading beyond Node v11
-FROM docker.io/node:11.15.0-alpine
+FROM docker.io/node:18.9.0-alpine
```

Comment:
Did you mean to upgrade this as well? Usually pull requests strive to only touch the pieces of code that are relevant to the work (this change seems a bit more contentious, and might break things are require package updates)

Reply:
Also curious whether package-lock.json needed to be updated :)

Reply:
Sorry, no idea. I am not a developer and not versed in using github. Simply went through the translation as a favour to one of the original devs.

Reply:
This is somewhat our fault actually. We merged some dependency updated (from dependabot) which we later had to revert. It looks like @SvenSt-BIE made his changes on top of those dependency updates prior to our reversion.